SparkPost/2web2ui
# AC-829: Changed invoice endpoints

### What Changed
 - Invoice endpoint updated to use billing endpoint

### How To Test
 - Go to /account/billing
 - Verify that invoices are populated correctly
 - Verify that invoices can be downloaded
